definitions/benefit/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -312,7 +312,7 @@
   {"LiveFrom": "30/07/2018", "ID": "ScannedDocument", "ListElementCode": "includeInBundle", "FieldType": "YesOrNo", "ElementLabel": "Include in bundle?", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "ScannedDocument", "ListElementCode": "originalSenderOtherPartyId", "FieldType": "Text", "ElementLabel": "Original sender other party ID", "FieldShowCondition": "type=\"AnyValueToFailThisCondition\"", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "ScannedDocument", "ListElementCode": "originalSenderOtherPartyName", "FieldType": "Text", "ElementLabel": "Original sender other party name", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/06/2019", "ID": "dwpResponseDocumentCT", "ListElementCode": "dwpResponseDocumentTitle", "FieldType": "Label", "ElementLabel": "Secretary of State's response", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/06/2019", "ID": "dwpResponseDocumentCT", "ListElementCode": "dwpResponseDocumentTitle", "FieldType": "Label", "ElementLabel": "Secretary of State's response", "SecurityClassification": "PUBLIC", "FieldShowCondition": "documentFileName=\"AnyValueToFailThisCondition\""},
```

Comment:
Wouldn't this hide it for all fields that use this complex types, which would mean it'll incorrectly hide the title from any case events that use this field?

Reply:
yes, it will hide it but only in the tabs
in the events we're setting up the title the appropriate  show/hide conditions
see [dwpChallengeValidity](https://github.com/hmcts/sscs-tribunals-case-api/blob/97c63e134466d3d10002f8256bb65a440955f61e/definitions/benefit/sheets/CaseEventToComplexTypes/CaseEventToComplexTypes-dwpChallengeValidity.json#L5), [hmctsResponseReviewed](https://github.com/hmcts/sscs-tribunals-case-api/blob/97c63e134466d3d10002f8256bb65a440955f61e/definitions/benefit/sheets/CaseEventToComplexTypes/CaseEventToComplexTypes-hmctsResponseReviewed.json#L5) and [dwpUploadResponse](https://github.com/hmcts/sscs-tribunals-case-api/blob/97c63e134466d3d10002f8256bb65a440955f61e/definitions/benefit/sheets/CaseEventToComplexTypes/CaseEventToComplexTypes-dwpUploadResponse.json#L5)